### DISCUSSION TITLE
JointJS  v3.7.7 Port position is always on the left

### DISCUSSION THREAD

**Discussion by ybarreto80** [clientIO/joint#2558] [Q&A]
### Introduction

Port position is always on the left.
ASPNET Core + Angularv17

```js
var port = {
  position: "right",
  label: {
    position: "left",
    markup: [{
      tagName: 'text',
      selector: 'label'
    }]
  },
  attrs: {
    portBody: {
      magnet: true,
      width: 16,
      height: 16,
      x: -8,
      y: -8,
      fill: '#03071E'
    },
    label: {
      text: 'in',
    }
  },
  markup: [{
    tagName: 'rect',
    selector: 'portBody'
  }]
};
```

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
The position should be defined for a `group` of ports, not for a single port.
Please refer to our [documentation](https://resources.jointjs.com/docs/jointjs/v4.0/joint.html#dia.Element.ports).
